docs/source/deployguide/use_CA.md

```diff
@@ -283,5 +284,7 @@ Where the folders and certificates are:
 
 Note that while the certificates themselves can be named anything you want, you should not change the name of the folders themselves, as Fabric expects to consume folders with certain names.
 
+Just as Node OUs make it no longer necessary to include a certificate of an admin in the organization MSP, it is not necessary to include the public certificate of a node admin to administer the node. **Any identity given a Node OU of `admin` by the CA listed in `config.yaml` can administer any of the nodes owned by that organization without needing to place the public certificate of that admin in the organization MSP or the local MSP**.
```

Comment:
Should we be more specific here:
Just as Node OUs make it no longer necessary to include a certificate of an admin in the organization MSP `admincerts` folder, it is not necessary to include the public certificate of a node admin to administer the node.

Also
**Any identity given a Node OU of `admin` by the CA listed in`config.yaml` `certificate` field